posthog/migrations/0466_datawarehousesavedquery_last_ran_at_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.14 on 2024-09-04 09:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0465_datawarehouse_stripe_account"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="datawarehousesavedquery",
+            name="last_ran_at",
+            field=models.DateTimeField(help_text="The timestamp at which this SavedQuery was last ran.", null=True),
+        ),
+        migrations.AddField(
+            model_name="datawarehousesavedquery",
+            name="last_run_status",
```

Comment:
Does this need a `last_run_` prefix? Can it just be `status`?

Reply:
Could be just status :+1: